datasource/src/main/resources/liquibase/changelog/changelog-master.xml

```diff
@@ -4,6 +4,6 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
 
-    <include file="changelog-0.1.xml" relativeToChangelogFile="true"/>
+    <include file="changelog-0.2.xml" relativeToChangelogFile="true"/>
```

Comment:
We should definetely add another changelog or add new changeset to the existing one.
@OlegKoshelev Once migrations are applied, they should never be removed / replaced / changed. If you want to change DB, you should create a new migration that should work on top of existing migrations, not instead of them.